packages/ruby/lib/readme/metrics.rb

```diff
@@ -121,34 +109,36 @@ def validate_options(options)
         raise Errors::ConfigurationError, Errors::BUFFER_LENGTH_ERROR
       end
 
-      if options[:development] && !is_a_boolean?(options[:development])
+      if options[:development] && !a_boolean?(options[:development])
         raise Errors::ConfigurationError, Errors::DEVELOPMENT_ERROR
       end
 
-      if options[:logger] && has_logger_inferface?(options[:logger])
+      if options[:logger] && logger_inferface?(options[:logger])
         raise Errors::ConfigurationError, Errors::LOGGER_ERROR
       end
     end
 
-    def has_logger_inferface?(logger)
-      [
-        :unknown,
-        :fatal,
-        :error,
-        :warn,
-        :info,
-        :debug
+    def logger_inferface?(logger)
+      %i[
+        unknown
+        fatal
+        error
+        warn
+        info
+        debug
       ].any? { |message| !logger.respond_to? message }
     end
 
-    def is_a_boolean?(arg)
-      arg == true || arg == false
+    def a_boolean?(arg)
+      [true, false].include?(arg)
     end
 
+    # rubocop:disable Style/InverseMethods
```

Comment:
Silencing this here because it gets confused on the `!user_info.values.any?(&:nil?)` line. It wants you to use `.none` instead of `.any` but if you change it to that then it wants you to use `.any` instead of `.none`. 🤷